### PR TITLE
Enable errorprone

### PR DESCRIPTION
**Pull request by al3xliu**
Yeah, this solutioin does work, I have tested it.

My references: 
https://stackoverflow.com/questions/56975581/how-to-setup-error-prone-with-gradle-getting-various-errors
https://github.com/google/error-prone/issues/864

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -39,13 +42,12 @@ dependencies {
 	}
 	implementation configurations.localDeps
 
-	compileOnly "com.google.errorprone:javac:9+181-r4173-1"
-
+	errorprone("com.google.errorprone:error_prone_core:2.3.3")
```
Comment (wmdietl) [inline build.gradle:45]: Isn't `2.11.0` the current version, according to https://mvnrepository.com/artifact/com.google.errorprone/error_prone_core ?

File: build.gradle [hunk 2]
```diff
@@ -39,13 +42,12 @@ dependencies {
...
+	errorproneJavac("com.google.errorprone:javac:${versions.errorproneJavacVersion}")
```
Comment (wmdietl) [inline build.gradle:46]: These two lines use parentheses `()` around the argument, whereas the next two don't. Can these be more uniform?

Comment (wmdietl) [inline build.gradle:46]:
```suggestion
	errorproneJavac("com.google.errorprone:javac:${versions.errorproneJavacVersion}")

```

File: build.gradle (hunk 1 above)
Comment (wmdietl) [reply build.gradle:45]: Should that also go into `ext.versions`, to put all configurations into one spot?

**Review by wmdietl — CHANGES_REQUESTED** [review]
Thanks!

File: build.gradle (hunk 2 above)
Comment (al3xliu) [reply build.gradle]: Thanks, fixed!

Comment (al3xliu) [reply build.gradle]: Thanks, fixed!

File: build.gradle (hunk 1 above)
Comment (al3xliu) [reply build.gradle]: Thanks, fixed!

Comment (wmdietl) [reply build.gradle]: I like option 2: use the newest version and ensure we're running on a compatible runtime.

**Comment by al3xliu** [comment]
Close and restart a new PR...

**Pull request closed without merging**